pkgs/development/python-modules/simperium3/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  requests,
+  setuptools,
+  fetchPypi,
+}:
+buildPythonPackage rec {
+  pname = "Simperium3";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchPypi {
```

```suggestion
  # No tags on the GitHub repo.
  src = fetchPypi {
```
It's better to explicitly add the reason why you are not using `fetchFromGitHub`.